src/gatsby/src/components/park/parkHeader.js

```diff
@@ -116,10 +118,12 @@ export default function ParkHeader({
                 ) : (
                   <p>Operating dates are unavailable.</p>
                 )}
-                <p>
-                  <a href="#park-camping-details-container">Check campgrounds</a> and{" "}
-                  <a href="#park-facility-container">facilities</a> for additional dates.
-                </p>
+                {(campings.length > 0 && facilities.length > 0) &&
```

Comment:
We should probably have 2 more conditions here:

Check campgrounds for additional dates.
`(campings.length > 0 && facilities.length === 0)`

Check facilities for additional dates.
`(campings.length === 0 && facilities.length > 0)`